src/sqs/queues.ts

```diff
@@ -9,12 +9,20 @@ const sqsQueues = {
 	backfill: new SqsQueue<BackfillMessagePayload>({ queueName: "backfill",
 		queueUrl: envVars.SQS_BACKFILL_QUEUE_URL,
 		queueRegion: envVars.SQS_BACKFILL_QUEUE_REGION,
-		longPollingIntervalSec: LONG_POLLING_INTERVAL_SEC}, backfillQueueMessageHandler),
+		longPollingIntervalSec: LONG_POLLING_INTERVAL_SEC,
+		timeoutSec: 10*60,
+		maxAttempts: 3
+	},
+	backfillQueueMessageHandler,
+	defaultErrorHandler
+	),
 
 	push: new SqsQueue<PushQueueMessagePayload>({ queueName: "push",
 		queueUrl: envVars.SQS_PUSH_QUEUE_URL,
 		queueRegion: envVars.SQS_PUSH_QUEUE_REGION,
-		longPollingIntervalSec: LONG_POLLING_INTERVAL_SEC}, pushQueueMessageHandler),
+		longPollingIntervalSec: LONG_POLLING_INTERVAL_SEC,
+		timeoutSec: 60,
+		maxAttempts: 5}, pushQueueMessageHandler,	defaultErrorHandler),
```

Comment:
Curious why maxAttempts/timeoutSec are different for push and backfill?

Reply:
Hence for Initial Sync queue we have many messages there anyway (duplicates), and with current logic they take much more time to process hence we make many calls to GitHub(especially when there is throttling)

So that's why it is different. For Redis queue we currently have the same configuration (except we don't retry messages)